bind/utils_test.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2017 The go-python Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package bind
+
+import (
+	"errors"
+	"testing"
+)
+
+func TestGetGoVersion(t *testing.T) {
+	var versionTests = []struct {
```

Comment:
no need to create a named variable for this list of tests.
just create that list inline of the `for` loop.

people are used to table-driven tests.

_i.e._:

```go
for _, tt := range []struct {
    info string
    ...
}{
   {"go1.5", 1, 5, nil},
   ...
}{
   major, minor, err := getGoVersion(tt.info)
   ...
}
```